services/src/main/java/org/keycloak/services/resources/ThemeResource.java

```diff
@@ -103,15 +103,23 @@ public Response getResource(@PathParam("version") String version, @PathParam("th
     }
 
     @GET
-    @Path("/{realm}/{theme}/{locale}")
+    @Path("/{realm}/{themeType}/{locale}")
```

Comment:
Should this not also include the theme at all times?
```suggestion
    @Path("/{realm}/{theme}/{themeType}/{locale}")
```